app/application/services/tsa_batch_queue.py

```
@@ -0,0 +1,50 @@
+"""In-memory queue for TSA batch anchoring (COMPLIANCE profile).
+
+The queue lives in application layer so both EventService and background
+workers can use it without introducing infrastructure dependencies.
+"""
+
+from __future__ import annotations
+
+import asyncio
+from dataclasses import dataclass
+from typing import List
```

Flake8 (7.3.0)

[error] 'typing.List' imported but unused

(F401)